package.json

```diff
@@ -35,7 +35,8 @@
     "start": "nebula serve --type sn-org-chart",
     "test:unit": "aw -c aw.config.js",
     "test:integration": "aw puppet --glob 'test/integration/**/*.spec.js' --chrome.headless true --chrome.slowMo 10",
-    "prepublishOnly": "rm -rf dist && npm run build"
+    "prepublishOnly": "rm -rf dist && npm run build",
+    "version": "node ./scripts/version-qext.js && git add -A"
```

Comment:
Potentially we can add a postversion script as well that pushes the changes and tag